design/sketch.webidl

```diff
@@ -820,7 +819,7 @@ enum GPUPowerPreference {
 };
 
 dictionary GPURequestAdapterOptions {
-    GPUPowerPreference powerPreference;
+    GPUPowerPreference powerPreference; // Intentionally no default value.
```

Comment:
The comment kiiiind of makes it sound like this is required (although it isn't in the IDL); maybe "Optional; unset means no preference."

Reply:
Yeah, +1 to optional.